ksql-engine/src/main/java/io/confluent/ksql/planner/PlanSourceExtractorVisitor.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2017 Confluent Inc.
```

Comment:
2018

Reply:
Yes, we need to update all of them. In a separate PR though :)